app/config/config.yml

```diff
@@ -60,6 +61,24 @@ framework:
           - mustache
           - twig
 
+hwi_oauth:
+    firewall_names:
+      - main
+    resource_owners:
+        elife:
+            type: oauth2
+            class: eLife\Journal\Security\OAuth2\ElifeResourceOwner
+            client_id: '%oauth2_client_id%'
+            client_secret: '%oauth2_client_secret%'
+            access_token_url: '%api_url%/oauth2/token'
+            authorization_url: '%api_url_public%/oauth2/authorize'
```

Comment:
Do we want the URI for OAuth to be `api.elifesciences.org/oauth2`? @giorgiosironi

Reply:
looks simple and standard

Reply:
forwarding api-gateway /oauth2 to profiles?

Reply:
Yep.